localstack/utils/tail.py

```diff
@@ -43,6 +43,10 @@ def close(self):
         self.started.clear()
         self.thread = None
 
+    def truncate_log(self):
+        with open(self.file_path, "wb") as outfile:
+            outfile.write(b"")
```

Comment:
nit: i think calling it `truncate_log` is incoherent with the level of abstraction that `FileListener` is on, which is much lower. could we just call it `truncate_file`?